.github/CODEOWNERS

```diff
@@ -12,6 +12,7 @@ shell.nix             @status-im/devops
 
 # Custom ownership for specific folders
 # Feel free to add yourself for any new packages you implement.
+/_docs/policies       @status-im/status-go-guild @iurimatias @alaibe @shivekkhurana @ilmotta
```

Comment:
I would add @plopezlpz here as lead owning chat protocols, which currently reside in status-go.

Reply:
Done